test/memory.carp

```diff
@@ -568,4 +594,5 @@
   (assert-no-leak test sumtype-12 "sumtype-12 does not leak")
   (assert-no-leak test box-1 "box-1 does not leak")
   (assert-no-leak test box-2 "box-2 does not leak")
+  (assert-no-leak test blit-1 "blit-1 does not leak")
```

Comment:
Do we need the opposite test (non-blit give error when moving twice) to make sure there is no regression or is that covered by existing tests?

Reply:
It should be covered.

Reply:
sg!